apps/template/src/App.tsx

```diff
@@ -39,20 +39,17 @@ export const App = (props: AppProps) => {
   })
 
   const fallbackRender = ({ error }: { error: { message: string } }) => {
-    return (
-      <div className="w-1/2">
-        <CodeBlock className={preErrorClasses} copy={false}>
-          {error?.message || error?.toString() || "An error occurred"}
-        </CodeBlock>
-      </div>
-    )
+    return <div className="w-1/2">{error?.message || error?.toString() || "An error occurred"}</div>
```

Comment:
I just could not figure out how to fix in this case I saw since this app does not really proper in productive I think it would be makes more sense to revert this changes and just ts-ignore them for now until we figure it out fully

Reply:
WDYT?

Reply:
Looks good reverted, thanks.